shesmu-server/src/main/java/ca/on/oicr/gsi/shesmu/Imyhat.java

```diff
@@ -200,6 +200,11 @@ public void streamJson(GeneratorAdapter method) {
 		public Object unpackJson(JsonNode node) {
 			return RuntimeSupport.stream(node.elements()).map(inner::unpackJson).collect(Collectors.toSet());
 		}
+
+		@Override
+		public String wdlType() {
```

Comment:
Throughout this section, make sure your labels are clear that we're not parsing wdl itself, but the JSON block that womtool produces that provides inputs for WDL workflows, and the 'types' are the string values inside that JSON block..

Reply:
Basically all the `wdlType` should be something like `wdlInputType`.

Reply:
Fixed.